src/components/activity-page/managed-interactive/iframe-runtime.tsx

```diff
@@ -121,6 +122,9 @@ export const IframeRuntime: React.FC<IProps> = forwardRef((props, ref) => {
       addListener("closeModal", (options: ICloseModal) => {
         closeModal(options);
       });
+      addListener("log", (logData: any) => {
+        Logger.log(logData.action, logData.data, logData.value, id, url);
+      });
```

Comment:
I think it'd be good if this function was following LARA implementation (provided the same fields or more), so we get the same logs no matter if we run LARA or AP:
https://github.com/concord-consortium/lara/blob/master/app/assets/javascripts/logger.js#L199-L207
but following the calls chain here, it seems it does, right?

Reply:
Yes, eventually the final message gets structured as it is structured in LARA (with extra fields added that were part of the requested additions in the story).  However, I added a commit that restructures the `Logger.log` function so the argument structure is the same as in LARA.  Hopefully it makes it easier to read and easier to compare to LARA.